# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: startupcache/StartupCache.cpp
# Commit: b6029c7c0016
# Full Hash: b6029c7c00166a24f5f89872baea83c24373cca0
# Author: Doug Thayer <dothayer@mozilla.com>
# Date: 2020-07-02 15:21:09
# Regressor Bug: 1627075
# File Overlap Count: 1
# Description:
#   Bug 1627075 - Build Omnijar file list from startup cache r=froydnj
#   
#   We would like to be able to defer opening the omnijar files until after startup
#   if the StartupCache has already been populated. Opening the omnijar files takes
#   a nontrivial time, at least on Windows, and almost everything in the omnijar
# ==============================================================================

diff -r 8ead55149095 -r b6029c7c0016 startupcache/StartupCache.cpp
--- a/startupcache/StartupCache.cpp	Thu Jul 02 02:59:21 2020 +0000
+++ b/startupcache/StartupCache.cpp	Thu Jul 02 02:49:31 2020 +0000
@@ -439,7 +439,7 @@
   return NS_OK;
 }
 
-// Makes a copy of the buffer, client retains ownership of inbuf.
+// Takes ownership of the input buffer
 nsresult StartupCache::PutBuffer(const char* id, UniquePtr<char[]>&& inbuf,
                                  uint32_t len) {
   NS_ASSERTION(NS_IsMainThread(),